modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -289,12 +294,12 @@ GLOBAL_VAR_INIT(successful_blood_chem, 0)
 
 /mob/living/simple_animal/cortical_borer/Life(delta_time, times_fired)
 	. = ..()
-	//can only do stuff when we are inside a human
-	if(!inside_human())
+	//can only do stuff when we are inside a LIVING human
+	if(!inside_human() || (human_host.stat == DEAD))
```

Comment:
you don't need the parenthesis around the second arg. you do however need to make sure there is a human host, because it will runtime with "null.stat" if there is no host.

Reply:
so maybe human_host?.stat == DEAD, or something like that

Reply:
Oddly enough it wasn't runtiming while outside of the host, I think inside_human might have caught that - but I changed it to human_host?.stat == DEAD and it seems fine.